src/main/java/uk/gov/hmcts/reform/sscs/ccd/service/CcdService.java

```diff
@@ -178,4 +178,8 @@ private CcdException logCcdException(String message, Exception ex) {
         log.error(message, ccdException);
         return ccdException;
     }
+
+    public void setSupplementaryData(IdamTokens idamTokens, Long caseId, Map<String, Map<String, Map<String, Object>>> supplementaryData) {
+        updateCcdCaseService.setSupplementaryData(idamTokens, caseId, supplementaryData);
```

Comment:
Could we not call the ccdClient directly?

Reply:
It could, was just following the existing pattern of code that does more